pkg/types/operation.go

```diff
@@ -53,19 +53,27 @@ const (
 	FAILED OperationState = "failed"
 )
 
+type RelatedType struct {
+	ID            string            `json:"id"`
```

Comment:
Isn't this should be omitempty also in the case where Criteria is used for identification?

Reply:
I was wondering for the same thing, but left it as is. May be it will be better to be omitempty, as it will be cleaner